### PR TITLE
remove universal conf block

### PR DESCRIPTION
**Pull request by earnubs**
Updates based on review of #74

### CODE REVIEW

File: test/unit/actions/t_customer.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ describe('async actions', () => {
   let scope;
 
   before(() => {
-    GLOBAL.MU_URL = conf.get('MU_URL');
+    global.MU_URL = conf.get('MU_URL');
```
Comment (earnubs) [inline test/unit/actions/t_customer.js]: "GLOBAL deprecated, use global" warning

File: server/server.js [hunk 2]
```diff
@@ -64,6 +64,11 @@ function serve(webpackIsomorphicTools) {
         };
         const store = configureStore(initialState);
 
+        // config we share from server side to client side
```
Comment (earnubs) [inline server/server.js]: You still need/want to import those settings into nconf, to allow a setting to be overridden from env vars, etc.

The only simple way I can think to do this, without ending up with 2 nconf instances, is make settings look like: 

```
{
  UNIVERSAL: {
  ...
  },
  SERVER: {
  ...
  }
}

 // nconf.get('UNIVERSAL:MU_URL');

```

But to keep symmetry in get signature the universal settings would need to be rendered into HTML as:

```
__CONFIG__ = {
  UNIVERSAL: conf.get('UNIVERSAL')
}

// conf.get('UNIVERSAL:MU_URL');
```

Which is close to what you're asking for I think?

Comment (earnubs) [inline server/server.js]: PS. in this PR, `settings.js` still defines everything, you just have to choose what is released to the browser explicitly in the render method of root (/). The proposal above is to make what is released to the browser implicit in the inclusion of it in `UNIVERSAL`

Comment (katiefenn) [inline server/server.js]: 👍

**Pull request merged**